src/appengine/handlers/jobs.py

```diff
@@ -24,10 +24,23 @@
 from datastore import ndb_utils
 from fuzzing import fuzzer_selection
 from handlers import base_handler
+from libs import filters
 from libs import form
 from libs import gcs
 from libs import handler
 from libs import helpers
+from libs.query import datastore_query
+
+PAGE_SIZE = 10
+MORE_LIMIT = 50 - PAGE_SIZE  # exactly 5 pages
+
+KEYWORD_FILTERS = [
+    filters.String('name', 'name'),
```

Comment:
Do we actually need this? I think we can skip this and keep things simple. We don't need to do property: value filtering for the name.

Reply:
yeah, we don't need this as all tokens will be in keywords. Thanks for pointing this out.